test/extended/router/scoped.go

```diff
@@ -262,13 +262,19 @@ func waitForRouterOKResponseExec(ns, execPodName, url, host string, timeoutSecon
 	return nil
 }
 
-func expectRouteStatusCodeRepeatedExec(ns, execPodName, url, host string, statusCode int, times int) error {
+func expectRouteStatusCodeRepeatedExec(ns, execPodName, url, host string, statusCode int, times int, proxy bool) error {
+	var extraArgs []string
+	if proxy {
+		extraArgs = append(extraArgs, "--haproxy-protocol")
```

Comment:
This argument isn't in the version of `curl` bundled in UBI...

Reply:
It's so old though... https://github.com/curl/curl/commit/6baeb6df35d24740c55239f24b5fc4ce86f375a5

In my dev clusters, the image that gets used is `gcr.io/kubernetes-e2e-test-images/agnhost:2.6` which has curl 7.61.1 (x86_64-alpine-linux-musl) libcurl/7.61.1 LibreSSL/2.5.5 zlib/1.2.11 libssh2/1.8.2

Reply:
https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/test/e2e/framework/pod/resource.go#L475